fplus-http-server/src/router/application.rs

```diff
@@ -311,26 +338,36 @@ pub async fn all_applications() -> impl Responder {
 #[get("/application/active")]
 pub async fn active(query: web::Query<GithubQueryParams>) -> impl Responder {
     let GithubQueryParams { owner, repo } = query.into_inner();
-    let apps = match LDNApplication::active(owner, repo, None).await {
-        Ok(app) => app,
-        Err(e) => return HttpResponse::BadRequest().body(e.to_string()),
-    };
-    HttpResponse::Ok().body(serde_json::to_string_pretty(&apps).unwrap())
+    match LDNApplication::active(owner, repo, None).await {
+        Ok(app) => match serde_json::to_string_pretty(&app) {
+            Ok(response) => HttpResponse::Ok().body(response),
+            Err(_) => HttpResponse::InternalServerError().body("Failed to parse success message"),
+        },
+        Err(e) => HttpResponse::BadRequest().body(e.to_string()),
+    }
```

Comment:
These matches are so unreadable, do we have a long-term task to replace them with `?`?

Reply:
I added a task to do this